skills/declarative/calendarSkill/dialogs/SearchEventsDialog/SearchEventsDialog.dialog

```diff
@@ -108,7 +112,8 @@
           "isFutureProperty": "=coalesce(dialog.futureEventsOnly, false)",
           "isFutureEventsOnlyProperty": "=dialog.futureEventsOnly",
           "dateTimeTypeProperty": "= coalesce(dialog.type, \"date\")",
-          "startProperty": "=coalesce(dialog.start, formatDateTime(utcNow(), 'yyyy-MM-dd'))"
+          "startProperty": "=coalesce(dialog.start, formatDateTime(addDays(utcNow(),-1), 'yyyy-MM-dd'))",
+          "orderProperty": "= dialog.order"
```

Comment:
Can we rename this to ordinal? That aligns with the LUIS entity naming.

Reply:
fixed